cmake/CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ if(NOT WIN32)
     set(onnxruntime_ENABLE_INSTRUMENT OFF)
   endif()
 else()
+  add_compile_definitions(WINVER=0x0601)  # Support Windows 7 and newer
```

Comment:
I think WINVER is controlled by CMAKE_SYSTEM_VERSION cmake variable.

Reply:
I'll investigate more and reach you back later.

Reply:
CMAKE_SYSTEM_VERSION points to the target SDK version. I'm pretty sure they are different things. Also, just searched through the CMake repo and there's no built-in variable for WINVER

Reply:
And adding an external /D define or a define on top of headers are the way recommended by the MS docs: https://docs.microsoft.com/en-us/windows/win32/winprog/using-the-windows-headers#setting-winver-or-_win32_winnt